bitnami/kong/templates/migrate-job.yaml

```
@@ -59,9 +59,9 @@ spec:
             - name: KONG_DATABASE
               value: {{ include "kong.database" . | quote }}
             {{- if (eq .Values.database "postgresql") }}
-            {{- if .Values.postgresql.auth.usePasswordFiles }}
+            {{- if .Values.usePasswordFiles }}
```

Shouldn't you use? This way, you take into account subchart parameters:
```suggestion
            {{- if or .Values.usePasswordFiles .Values.postgresql.auth.usePasswordFiles }}
```
This also apply to Cassandra

---

All right! I'll buy you that